sources/packages/backend/apps/api/src/route-controllers/report/_tests_/e2e/report.aest.controller.exportReport.e2e-spec.ts

```diff
@@ -147,4 +148,97 @@ describe("ReportAestController(e2e)-exportReport", () => {
         );
       });
   });
+
+  it("Should generate the Institution Designation report when a report generation request is made with the appropriate date range filters.", async () => {
+    // Arrange
+    const [
+      approvedDesignationAgreementLocation,
+      declinedDesignationAgreementLocation,
+    ] = await saveFakeDesignationAgreementLocation(db, {
+      numberOfLocations: 2,
+      initialValues: {
+        endDate: getISODateOnlyString(addDays(30, new Date())),
+      },
+    });
+    const designationAgreement =
+      approvedDesignationAgreementLocation.designationAgreement;
+    const designatedInstitutionLocation =
+      approvedDesignationAgreementLocation.institutionLocation;
+    const nonDesignatedInstitutionLocation =
+      declinedDesignationAgreementLocation.institutionLocation;
+    const payload = {
+      reportName: "Institution_Designation_Report",
+      params: {
+        startDate: new Date(),
+        endDate: addDays(31, new Date()),
+      },
+    };
+    const dryRunSubmissionMock = jest.fn().mockResolvedValue({
+      valid: true,
+      formName: FormNames.ExportFinancialReports,
+      data: { data: payload },
+    });
+    formService.dryRunSubmission = dryRunSubmissionMock;
+    const endpoint = "/aest/report";
+    const ministryUserToken = await getAESTToken(
+      AESTGroups.BusinessAdministrators,
+    );
+
+    // Act/Assert
+    await request(app.getHttpServer())
+      .post(endpoint)
+      .send(payload)
+      .auth(ministryUserToken, BEARER_AUTH_TYPE)
+      .expect(HttpStatus.CREATED)
+      .then((response) => {
+        const fileContent = response.request.res["text"];
+        const parsedResult = parse(fileContent, {
+          header: true,
+        });
+        expect(parsedResult.data).toEqual(
+          expect.arrayContaining([
+            {
+              "Institution Operating Name":
+                designatedInstitutionLocation.institution.operatingName,
+              "Location Name": designatedInstitutionLocation.name,
+              "Location Code": designatedInstitutionLocation.institutionCode,
+              "Institution Type": "BC Private",
+              "Designation Status": designationAgreement.designationStatus,
+              "Assessed Date": "",
+              "Expiry Date": designationAgreement.endDate,
+              "Request for designation":
+                approvedDesignationAgreementLocation.requested.toString(),
+              "Approved for designation":
+                approvedDesignationAgreementLocation.approved.toString(),
+              "Location Contact":
+                designatedInstitutionLocation.primaryContact.firstName +
```

Comment:
not a blocker as designatedInstitutionLocation.primaryContact is repeated can create a const, same for nonDesignatedInstitutionLocation.primaryContact